torch/_inductor/codegen/wrapper.py

```diff
@@ -1379,6 +1477,56 @@ def wrap_arg(arg):
 
         return device_index, call_args
 
+    def generate_example_arg_value(self, arg, arg_type=None, raw_arg=None, index=None):
```

Comment:
We do not need to call `torch.rand` for outputs - can just can just call empty_strided